packages/main/src/TabContainer.js

```diff
@@ -226,6 +227,46 @@ const metadata = {
 			type: String,
 		},
 
+		/**
+		 * Defines the background color of the Tab Container's header.
+		 *
+		 * <br><br>
+		 * Available options are:
+		 * <ul>
+		 * <li><code>Solid</code></li>
+		 * <li><code>Transparent</code></li>
+		 * </ul>
+		 *
+		 * @type {sap.ui.webcomponents.main.types.TabContainerBackgroundDesign.prototype}
+		 * @defaultvalue "Solid"
+		 * @since 1.10.0
+		 * @public
+		 */
+		headerBackgroundDesign: {
+			type: TabContainerBackgroundDesign,
+			defaultValue: TabContainerBackgroundDesign.Solid,
+		},
+
+		/**
+		 * Defines the background color of the Tab Container's content.
+		 *
+		 * <br><br>
```

Comment:
Respectively here:
* Sets the background color of the Tab Container's content as <code>Solid</code>, <code>Transparent</code>, or <code>Translucent</code>.